devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -233,7 +233,7 @@ private getSupportedButtonValues() {
 private getModelNumberOfButtons() {[
 		"3450-L" : 4,
 		"3450-L2" : 4,
-		"SceneSwitch-EM-3.0" : 4
+		"SceneSwitch-EM-3.0" : 3
```

Comment:
I think this should be left at 4 because it's a 4 button device.

Reply:
I've checked zigbee scene keypad DTH, for 4 buttons scene keypad, there is 1 main button and 3 child buttons, when create child buttons, index are from 2 to 4. since you requested me to start the index from 1, I have to change it to 3

and from the function
private addChildButtons(numberOfButtons) {
for(def endpoint : 1..numberOfButtons) {
...}

it seems the value is the number of child buttons